src/plugins/home/public/application/components/home_app.js

```diff
@@ -110,6 +106,9 @@ export function HomeApp({ directories, solutions }) {
               telemetry={telemetry}
             />
           </Route>
+          <Route exact path="/">
+            <Homepage olly={isOlly} />
```

Comment:
ideally `olly` reference would be removed here, but that can be a followup since that was not caused by this PR.